### PR TITLE
add TIM tag to feed integrations

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24069

## Description
Check if a pack is a TIM pack by:
Has a playbook that starts with "TIM" or has an integration with a "feed" parameter.
If so, make sure it has the TIM tag.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Tests/Marketplace/marketplace_services.py [hunk 1]
```diff
@@ -330,6 +344,24 @@ def _get_all_pack_images(pack_integration_images, display_dependencies_images, d
...
+        Checks if an integration is a feed integration. If so, updates Pack._is_feed
+        Args:
+            yaml_content: The yaml content extracted by yaml.safe_load().
+            yaml_type: The type of object to check. Should be 'Playbook' or 'Integration'.
+
+        Returns:
+            Doesn't return
+        """
+        if yaml_type == 'Integration':
+            for configuration in yaml_content.get('configuration'):
```
Comment (IkaDemisto) [inline Tests/Marketplace/marketplace_services.py:358]: Why are you searching in `configuration` section in the yml and not under `script` -> `feed` == true?

File: Tests/Marketplace/Tests/marketplace_services_test.py [hunk 2]
```diff
@@ -236,6 +248,32 @@ def test_get_higher_server_version(self, current_string_version, compared_conten
 
         assert result == expected_result
 
+    @pytest.mark.parametrize('yaml_context, yaml_type, is_actually_feed',
+                             [
+                                 # Check is_feed by Integration
+                                 ({'category': 'TIM', 'configuration': [{'display': 'Services', 'name': 'url',
```
Comment (IkaDemisto) [inline Tests/Marketplace/Tests/marketplace_services_test.py:254]: Same question about the feed integration yml feed boolean.

File: Tests/Marketplace/Tests/marketplace_services_test.py [hunk 3]
```diff
@@ -90,6 +90,18 @@ def test_parsed_metadata_with_price(self, pack_metadata_input, expected, mocker)
 
         assert parsed_metadata['price'] == expected
 
+    @pytest.mark.parametrize('is_feed_pack, tags',
+                             [(True, ["tag number one", "Tag number two", 'TIM']),
+                              (False, ["tag number one", "Tag number two"])
+                              ])
+    def test_tim_tag_added_to_feed_pack(self, dummy_pack_metadata, is_feed_pack, tags):
+        parsed_metadata = Pack._parse_pack_metadata(user_metadata=dummy_pack_metadata, pack_content_items={},
```
Comment (IkaDemisto) [inline Tests/Marketplace/Tests/marketplace_services_test.py:98]: Please add doc string that explains the added tests with the common format:
```
        Scenario: 
        Given
        When
        Then
```

File: Tests/Marketplace/marketplace_services.py (hunk 1 above)
Comment (orlichter1) [reply Tests/Marketplace/marketplace_services.py:358]: @IkaDemisto no actual reason. I didn't know this field. I just knew that every feed integration has a `Fetch Indicators` command. Will change to `script`->`feed` check.

**Pull request merged**